dockerfiles/Dockerfile.ci

```diff
@@ -1,31 +1,18 @@
-FROM docker/compose:1.24.0 as compose
-FROM ruby:2.7-alpine as ci
+FROM ubuntu:20.04 as ci
 
-ENV GLIBC_VERSION 2.28-r0
+RUN apt-get update
+RUN apt-get install -y ruby-full apt-utils docker-compose wget unzip bash bundler
```

Comment:
Is there a reason to use this over composing the image from both `ubuntu` and `docker/compose` images?

Reply:
To clarify - installing `docker-compose` this way

Reply:
It just felt like the natural way of doing it - start from a base image and install what we need.  My assumption is that the original approach was just a workaround for there not being a package available for Alpine and I think this yields an overall cleaner result.